scripts/ruby/lib/file_type.rb

```diff
@@ -129,7 +130,6 @@ def transform_solr
     else
       req = exec_xsl(@file_location, @script_solr, "xml", nil, @options["variables_solr"])
     end
-    @solr_req if req && req.has_key?("doc")
```

Comment:
I was wondering what this was but just made a note to see if it was touched in later commits. And it was. :-D
I think things look good!